### ISSUE TITLE
self-signed certificate error

### ISSUE THREAD

**Issue by arefaydi** [nextcloud/circles#212]
While I share a file with a circle, this error occurs :
**_cURL error 60: SSL certificate problem: self signed certificate_**
File shared successfully but it didn't shown on the share tab until reload page. SSL enabled with self signed certificate.

**Comment by DanieleCir** [comment]
I can confirm the same issue, when I try to share with a Circle_group.
Version in use: NC-14 all updated from "stable channel".
I'm using a regualar/valid certificate got from GlobalSign (not self signed); it is a wildcard certificate, used for:  *.domain.ext
The server I created is called like  nc.**domain.ext**

I just added:
[curl]
curl.cainfo = /etc/curl_ca/cacert.pem
to my php.ini after I downloaded the most updated cacert.pem from this link: https://curl.haxx.se/ca/cacert.pem

But the problem is still there.

Any suggestion?

Thank you.
Daniele
